docs/blog/2018-08-09-swag-store/index.md

```diff
@@ -18,15 +18,14 @@ We’ve spent the last couple weeks working the kinks out, and we couldn’t be
     src="./images/gatsby-swag-twitter.jpg"
   />
   <figcaption>
-    Tweets from{" "}
-    <a href="https://twitter.com/DeaNHtiD99/status/1023204484183416832">Dean</a>,{" "}
-    <a href="https://twitter.com/ARebelBelle/status/1020044426712735744">
+    {/* prettier-ignore */}
```

Comment:
hmm, what happens here if we don't put the prettier-ignore?

Reply:
Prettier is unable to fix code style issues with this block.
`npx prettier --write docs/blog/2018-08-09-swag-store/index.md` always switches this block in these two versions but `--check` always fails.
<details>
<summary>Versions</summary>

Version 1
``` 
    Tweets from{" "}
    <a href="https://twitter.com/DeaNHtiD99/status/1023204484183416832">Dean</a>
    ,{" "}
    <a href="https://twitter.com/ARebelBelle/status/1020044426712735744">
      Elle
    </a>
    ,{" "}
    <a href="https://twitter.com/chaseadamsio/status/1021896138503245824">
      Chase
    </a>
    , and <a href="https://twitter.com/kato_katherine/status/1021967765400211456">
      Katherine
    </a> showing off their new Gatsby swag.
```

Version 2
```
    Tweets from{" "}
    <a href="https://twitter.com/DeaNHtiD99/status/1023204484183416832">Dean</a>,{" "}
    <a href="https://twitter.com/ARebelBelle/status/1020044426712735744">
      Elle
    </a>,{" "}
    <a href="https://twitter.com/chaseadamsio/status/1021896138503245824">
      Chase
    </a>
    , and <a href="https://twitter.com/kato_katherine/status/1021967765400211456">
      Katherine
    </a> showing off their new Gatsby swag.
```
</details>

Reply:
Hmmm that's annoying. it sucks that MDX can't treat inner content as markdown. I'll talk to the team about it and see if we can come up with a solution but otherwise I guess we'll have to leave it at this.

Reply:
Okay apparently there _is_ a way to do it, by leaving a blank line in between the tags:

```
<Button>

Here is a [button](https://github.com)

</Button>
```

Unfortunately this seems to break Prettier's parsing:

https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAZgSwOYFcBOcAfADpRkASchABJgM40CGNA2gDaZQDWAugBQB6AEZNRogJRkyqQVjyEiIADQgIABxiZo9ZKCb58EAO4AFAwl0omANwiYAJipDD8TMNzgwAyuvddsZBh8XDhVAAsYAFt2AHVwzHh6PzA4b0tEzBtEgE9kcHpdVS56ahhTN2wopmR0JnZS1QAregAPACE3Dy9vJii4ABkuOFr6xpAW1u8A9jgARVwIeFGGsJA-fFL8fKiHVud1fC4YWMcYcOQADgAGVUOIUti3dXyoOBtqZwBHRfgKjSsICY9AAtG84A4Ic5CD9MIQKkwqjUkHVVqpSlFMEEQmt6DN5r8RiixmsYGJTg5zsgAEyqYJMTCcKDYADCECi1Ve0BGqlwpQAKmIrKjxjZQgBJKCQ2DeMBHTQAQSl3hgOVmK1KAF9NUA